### DISCUSSION TITLE
Saving changes to actions.json

### DISCUSSION THREAD

**Discussion by Gregory1701** [codefori/vscode-ibmi#1278] [IBM i] [answered]
Hey guys,
In this video, I want to add a compile command for CLLE types.  But when I make the change and save it, the next time I reopen actions.json my change is not there.  What am I doing wrong?
![2023-05-09_07-46-12](https://github.com/halcyon-tech/vscode-ibmi/assets/132368707/f40567e7-64b0-4dc1-9e8f-d5083d600cf9)

**Comment by worksofliam** [comment, marked as answer]
Hey @Gregory1701 - launch actions setup overwrites the file every time you run in. I generally recommend only running it once with all the options you require and then adding your own after.
